vault_cli/cli.py

```diff
@@ -183,18 +183,28 @@ def list_(client_obj: client.VaultClientBase, path: str):
 
 
 @cli.command(name="get-all")
+@click.option(
+    "--flat",
```

Comment:
I'd rather have `--flat/--nested` which is far easier to change later. Plus, with this syntax, is_flag is implied.

Reply:
Better yet, `--nested/--flat` so that your variable will be named "nested", which feels more in line with what we're going to do.

Reply:
I don't want people to use the `--nested` flag because I think we won't change the default in the future but remove the "nested" output

Reply:
Oh, I saw changing the defaut as part of the deprecation path.

Reply:
fixed